> ```
> ##[error](node(0,0): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
>  (Use `node --trace-warnings ...` to show where the warning was created)
> ```

I'm wondering if rerunning with https://github.com/dotnet/aspnetcore/pull/41938 will make a difference otherwise we might have to look into changing how we do this:

--- a/src/SignalR/clients/ts/signalr/package.json
+++ b/src/SignalR/clients/ts/signalr/package.json
@@ -52,7 +52,7 @@
   },
   "dependencies": {
     "abort-controller": "^3.0.0",
-    "eventsource": "^2.0.0",
+    "eventsource": "^2.0.2",
     "fetch-cookie": "^2.0.3",
     "node-fetch": "^2.6.7",
     "ws": "^7.4.5"

--- a/src/SignalR/clients/ts/signalr/package.json
+++ b/src/SignalR/clients/ts/signalr/package.json
@@ -44,7 +44,7 @@
     "src/**/*"
   ],
   "devDependencies": {
-    "@types/eventsource": "^1.1.5",
+    "@types/eventsource": "^1.1.8",
     "@types/jest": "^26.0.20",
     "@types/node": "^14.14.31",
     "@types/tough-cookie": "^4.0.0",
@@ -52,8 +52,8 @@
   },
   "dependencies": {
     "abort-controller": "^3.0.0",
-    "eventsource": "^1.0.7",
-    "fetch-cookie": "^0.11.0",
+    "eventsource": "^2.0.0",
+    "fetch-cookie": "^2.0.3",
     "node-fetch": "^2.6.7",
     "ws": "^7.4.5"
   },

--- a/src/SignalR/clients/ts/signalr/package.json
+++ b/src/SignalR/clients/ts/signalr/package.json
@@ -58,7 +58,6 @@
     "ws": "^7.4.5"
   },
   "resolutions": {
-    "ansi-regex": "5.0.1",
-    "url-parse": ">=1.5.6"
+    "ansi-regex": "5.0.1"
   }
 }

